docs/source/actions/custom-action.rst

```diff
@@ -13,3 +13,45 @@ the `run_action` method.
         def run_action(self):
             # Include here the logic of your action
             # (...)
+
+
+Fallback Actions
+================
+
+When creating your own custom actions, you can also add a fallback action to run if
+an action throws an unhandled exception. To do this, add a fallback attribute to
+your custom action.
+
+.. code-block:: python
+
+    from spidermon.core.actions import Action
+
+    class MyFallbackAction(Action):
+        def run_action(self):
+            # Include here the logic of your action
+            # Runs if MyCustomAction().run_action() throws an unhandled exception
+            # (...)
+
+    class MyCustomAction(Action):
+        fallback = MyFallbackAction()
+        def run_action(self):
+            # Include here the logic of your action
+            # (...)
+
+
+You can also add fallbacks to spidermon built-in actions by subclassing them.
+
+.. code-block:: python
+
+    from spidermon.core.actions import Action
+    from spdiermon.contrib.actions import Sentry
+
+    class MyFallbackAction(Action):
+        # Runs if MyCustomSentryAction fails
+        def run_action(self):
+            # Include here the logic of your action
+            # (...)
+
+    class MyCustomSentryAction(Sentry):
+        fallback = MyFallbackAction()
```

Comment:
**suggestion**

Instead of an instance of the action, imo it would be better to have just the class or the path of the class and instantiate it only if when used:

```
    class MyCustomSentryAction(Sentry):
        fallback = MyFallbackAction
```
or 
```
    class MyCustomSentryAction(Sentry):
        fallback = "myproject.actions.MyFallbackAction"
```

This would avoid that initialization code of the custom action (for example, if `__init__` of the action do something) be made without necessity.